src/utils/log2_fold_change.rs

```diff
@@ -0,0 +1,49 @@
+use std::ops::Not;
+
+use ordered_float::NotNan;
+
+use crate::variants::model::AlleleFreq;
+use crate::utils::comparison::Comparison;
+
+#[derive(Debug, Clone)]
+pub(crate) struct Log2FoldChange {
+    vaf_a: AlleleFreq,
+    vaf_b: AlleleFreq,
+    l2fc: f64,
+}
+
+impl Log2FoldChange {
+    pub(crate) fn new(vaf_a: AlleleFreq, vaf_b: AlleleFreq) -> Self {
+        Self {
+            vaf_a,
+            vaf_b,
+            l2fc: vaf_a.log2() - vaf_b.log2(),
+        }
+    }
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub(crate) struct Log2FoldChangePredicate(pub(crate) Comparison, pub(crate) NotNan<f64>);
```

Comment:
Rather use a struct and two fields `comp` and `value` here? Do we need Ord and PartialOrd (does not make sense to me)?

Reply:
Ord and PartialOrd are needed because this is a requirement for the some stuff further up the chain.